src/mock-data/fiat-account.ts

```diff
@@ -44,7 +45,7 @@ const pixAccount: PostFiatAccountRequestBody = {
     accountName: 'My PIX Account',
     fiatAccountType: FiatAccountType.BankAccount,
     keyType: PIXKeyTypeEnum.RANDOM,
-    key: 'a'.repeat(32),
+    key: crypto.randomUUID(),
```

Comment:
I'd prefer if the mock data was fixed, rather than different every time you run the test

Reply:
```suggestion
    key: 'a332fb0d-63d7-4cf6-af37-c607d9618714',  // arbitrary uuid
```